mods/voidmulti.sc2mod/base.sc2data/gamedata/effectdata.xml

```diff
@@ -2157,6 +2159,13 @@
         <Amount value="5"/>
         <Kind value="Ranged"/>
     </CEffectDamage>
+    <CEffectApplyBehavior id="BatteryCooldownAB">
+        <EditorCategories value=""/>
+        <WhichUnit Value="Source"/>
+        <Behavior value="BatteryAcquireTargetCooldown"/>
+        <Flags index="UseDuration" value="1"/>
+        <Duration value="0.063"/>
+    </CEffectApplyBehavior>
```

Comment:
Fixed an issue where repeatedly issuing a Stop command could increase the shield regen rate of a Shield Battery's target

Adds the Cooldown behavior to the Shield Battery.

Does the duration override change anything concerning stacks like resetting the duration? This is the exact duration the behavior already has.
I suspect that use duration and duration can be removed.